demo/api/src/products/entities/product.entity.ts

```diff
@@ -95,70 +81,108 @@ export class Product extends BaseEntity<Product, "id"> {
         sort: true, //default is true
         input: true, //default is true
     })
+    @CsvColumn("title")
+    @IsString()
     title: string;
 
     @Enum({ items: () => ProductStatus })
     @Field(() => ProductStatus)
+    @CsvColumn("status")
+    @IsEnum(ProductStatus)
     status: ProductStatus = ProductStatus.Unpublished;
 
     @Property()
     @Field()
+    @CsvColumn("slug")
+    @IsString()
     slug: string;
 
     @Property()
     @Field()
+    @CsvColumn("long description")
+    @IsString()
     description: string;
 
     @Enum({ items: () => ProductType })
     @Field(() => ProductType)
+    @CsvColumn("type")
+    @IsEnum(ProductType)
     type: ProductType;
 
     @Field(() => [ProductType])
     @Enum({ items: () => ProductType, array: true })
+    @CsvColumn("additionalTypes")
+    @IsArray()
+    @IsEnum(ProductType, { each: true })
+    @FieldTransformer((value: string) => (value ? value.split(",").map((type) => ProductType[type.trim() as keyof typeof ProductType]) : []))
```

Comment:
This basically does the same like `@Transform()` from class-transformer, which will be called when using plainToInstance -> can we use that instead?